rust/tombi-cli/src/app/command/completion.rs

```diff
@@ -1,16 +1,12 @@
+use std::io;
+
 use clap::CommandFactory;
 
-/// Generate shell completion.
-#[derive(clap::Args, Debug)]
-pub struct Args {
-    /// Shell to generate completion for
-    #[arg(value_enum)]
-    shell: clap_complete::Shell,
-}
+use crate::args::{Args, CompletionArgs};
```

Comment:
As a personal preference, I keep commands and their arguments in the same file.

Please submit a PR containing only the parts related to `man` 🙏